arch/risc-v/src/common/riscv_swint.c

```diff
@@ -223,6 +227,18 @@ int riscv_swint(int irq, void *context, void *arg)
         {
           DEBUGASSERT(regs[REG_A1] != 0);
           CURRENT_REGS = (uintptr_t *)regs[REG_A1];
+#ifdef CONFIG_ARCH_ADDRENV
+
+          tcb = this_task();
```

Comment:
I get warning about implicit declaration of `this_task()`. 

Either ` #include "sched/sched.h"`

or change to `tcb = nxsched_self();`

Reply:
Emm, to minimal the change of riscv_swint.c, now we call `group_addrenv(NULL)` instead.